test/evergreen/configure_combinations.sh

```diff
@@ -64,8 +66,23 @@ for flag in $flags ; do
                        echo "*** ERROR: $flag, $option"
                fi
                IFS="$cr_IFS"
-               echo hello
        done
 done
 IFS=$saved_IFS
-exit $ecode
+if [ $ecode -eq 1 ]; then
+        exit 1
+fi
+
+# Handle special build combination for running all the diag tests.
+flag=`CFLAGS=\"-g -Werror\"`
```

Comment:
Can't you merge this into build test?

Reply:
I discussed it with @lukech. The main difference is `make` vs `make all`, and, `make -C examples/c check` vs `make check`. The special case test is more comprehensive compared to previous ones.